gemrb/plugins/SDLVideo/CMakeLists.txt

```diff
@@ -1,13 +1,21 @@
-INCLUDE_DIRECTORIES( ${SDL_INCLUDE_DIR} ${GLEW_INCLUDE_DIR} )
+INCLUDE_DIRECTORIES( ${SDL_INCLUDE_DIR})
 
 SET(COMMON_FILES COCOA SDLVideo.cpp SDLSurfaceSprite2D.cpp)
 IF(SDL_BACKEND STREQUAL "SDL2")
-	IF(OPENGL_BACKEND)
+	IF(NOT OPENGL_BACKEND STREQUAL "None")
 		ADD_GEMRB_PLUGIN( SDLVideo ${COMMON_FILES} SDL20Video.cpp GLSLProgram.cpp)
-		IF(APPLE)
-			TARGET_LINK_LIBRARIES( SDLVideo ${SDL_LIBRARY} ${OPENGL_LIBRARY} ${CMAKE_THREAD_LIBS_INIT} ${COCOA_LIBRARY_PATH})
+		target_compile_definitions( SDLVideo PRIVATE USE_OPENGL_BACKEND)
+		target_compile_definitions( SDLVideo PRIVATE USE_$<UPPER_CASE:${OPENGL_BACKEND}_API>)
+		TARGET_LINK_LIBRARIES( SDLVideo ${SDL_LIBRARY} ${CMAKE_THREAD_LIBS_INIT} ${COCOA_LIBRARY_PATH})
+
+		IF(OPENGL_BACKEND STREQUAL "OpenGL")
+			IF(APPLE)
+				TARGET_LINK_LIBRARIES( SDLVideo ${OPENGL_LIBRARIES})
+			ELSE()
+				TARGET_LINK_LIBRARIES( SDLVideo ${OPENGL_LIBRARIES})
```

Comment:
This looks redundant now. Previously the only difference was in glew, but now you removed it ...

Reply:
Indeed - fixed.